src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/UserPersonDetail.kt

```diff
@@ -30,7 +30,7 @@ data class UserPersonDetail(
   @OneToMany(fetch = FetchType.LAZY)
   @JoinColumn(name = "USERNAME", updatable = false, insertable = false, nullable = false)
   @Where(clause = "CASELOAD_ID = 'NWEB'")
-  val dpsRoles: List<UserCaseloadRole> = listOf(),
+  val dpsRoles: MutableList<UserCaseloadRole> = mutableListOf(),
 
```

Comment:
We see if I can move back to using a immutable list, but need to this to create test data